content/dstu2/medication-statement.md

```diff
@@ -26,3 +30,106 @@ Search for MedicationStatements that meet supplied query parameters:
 
 <%= headers 200 %>
 <%= json(:dstu2_medication_statement_bundle) %>
+
+## Create
+
+Create new MedicationStatement. The medicationReference field must refer to a contained Medication with the code field populated.
```

Comment:
Forgot to mention this in our meeting, but you can only create in-progress med statements.